src/sgame/components/HealthComponent.cpp

```diff
@@ -119,11 +116,32 @@ Util::optional<Vec3> direction, int flags, meansOfDeath_t meansOfDeath) {
 
 	float take = amount;
 
+	// halve team damage according to the team, ignoring buildings
```

Comment:
Comment shouldn't say "halve" since it's configurable?

Reply:
updated